train.py

```diff
@@ -128,6 +128,7 @@
 
 def main():
     """main"""
+    # pylint: disable=too-many-branches
```

Comment:
please remove and fix limits in .pylintrc and .github/linters/.python-lint

Reply:
Realized in the latest commit.